src/System.Runtime.Serialization.Xml/ref/project.json

```diff
@@ -3,7 +3,7 @@
     "System.Runtime": "4.0.0",
     "System.Xml.ReaderWriter": "4.0.0",
     "System.IO": "4.0.0",
-    "System.Runtime.Serialization.Primitives": "4.0.10",
+    "System.Runtime.Serialization.Primitives": "4.1.0-beta-*",
```

Comment:
We've added a new interface to System.Runtime.Serialization.Primitives contract. From discussion we've agreed to take dependency of the new S.R.S.Primitives package.